hypervisor/src/hypervisor/partition.rs

```diff
@@ -128,76 +173,87 @@ impl Run {
+                    FileMounter {
+                        source: None,
+                        target: "".into(),
+                        fstype: Some("tmpfs".into()),
+                        flags: MsFlags::empty(),
+                        data: Some("size=500k".into()),
+                        is_dir: true,
+                    },
+                    // Mount binary
+                    FileMounter {
+                        source: Some(base.bin.clone()),
+                        target: "bin".into(),
+                        fstype: None,
+                        flags: MsFlags::MS_RDONLY | MsFlags::MS_BIND,
+                        data: None,
+                        is_dir: false,
+                    },
+                    // Mount /dev/null (for stdio::null)
+                    FileMounter {
+                        source: Some("/dev/null".into()),
+                        target: "dev/null".into(),
+                        fstype: None,
+                        flags: MsFlags::MS_RDONLY | MsFlags::MS_BIND,
+                        data: None,
+                        is_dir: false,
+                    },
+                    // Mount proc
+                    FileMounter {
+                        source: Some("/proc".into()),
+                        target: "proc".into(),
+                        fstype: Some("proc".into()),
+                        flags: MsFlags::empty(),
+                        data: None,
+                        is_dir: true,
+                    },
+                    // Mount CGroup v2
+                    FileMounter {
+                        source: None,
+                        target: "sys/fs/cgroup".into(),
+                        fstype: Some("cgroup2".into()),
+                        flags: MsFlags::empty(),
+                        data: None,
+                        is_dir: true,
+                    },
+                ];
+
+                for m in &base.mounts {
+                    let source = m.0.clone();
+                    let mut target = m.1.clone();
+
+                    if !source.exists() {
+                        panic!("File/Directory {} not existent", m.0.display());
+                    }
+
+                    if target.is_absolute() {
+                        // Convert absolute paths into relative ones.
+                        // Otherwise we will receive a permission error.
+                        // TODO: Make this a function?
+                        target = target.strip_prefix("/").unwrap().into();
+                        assert!(target.is_relative());
+                    }
+
+                    mounts.push(FileMounter {
+                        source: Some(source),
+                        target: target,
+                        fstype: None,
+                        flags: MsFlags::MS_BIND,
+                        data: None,
+                        is_dir: m.0.is_dir(),
+                    });
+                }
```

Comment:
You should consider implementing [`TryFrom`](https://doc.rust-lang.org/std/convert/trait.TryFrom.html) for our `FileMounter`. Also this conversion could be done at config-parse-time. Returning an error as early as possible if for example the source file does not exist.
```Rust
impl TryFrom<(PathBuf, PathBuf)> for FileMounter{
    ...
}
```

Reply:
I will take a look at this.